src/commands/crashes/category.txt

```diff
@@ -0,0 +1 @@
+Commands related to Crashes
```

Comment:
@anlinde this file contains the help for the category, not just the one command. Is there better text for the set of commands as a whole? Or are you only ever going to have the one command?